ReactWindows/ReactNative/UIManager/BaseViewManager.cs

```diff
@@ -280,6 +312,77 @@ private void OnPointerExited(object sender, PointerRoutedEventArgs e)
             TouchHandler.OnPointerExited(view, e);
         }
 
+        // This event is intentionally allowed to bubble up, so all the corresponding
+        // RN views get the proper enter-leave event pairs.
+        private static async void OnDragEnter(object sender, DragEventArgs args)
+        {
+            if (sender is TFrameworkElement view)
+            {
+                var data = await GetDataTransferInfo(args.DataView);
+
+                view.GetReactContext()
+                    .GetNativeModule<UIManagerModule>()
+                    .EventDispatcher
+                    .DispatchEvent(new DragDropEvent(view.GetTag(), "topDragEnter", data));
+            }
+        }
+
+        private static async void OnDragOver(object sender, DragEventArgs args)
+        {
+            if (sender is TFrameworkElement view)
+            {
```

Comment:
I think we have some guarantees that the sender is TFrameworkElement, we could just do a cast here. Same for other event handlers.

Reply:
OK, fixed.